WebVTT: Add missing default css style for text and backgound colors.

pgorszkowski opened the pull request:
#### df2ca59a2ba44f9edc76105314c544b0933f3365
<pre>
WebVTT: Add missing default css style for text and backgound colors.
<a href="https://bugs.webkit.org/show_bug.cgi?id=238540">https://bugs.webkit.org/show_bug.cgi?id=238540</a>

Base on specification from:
<a href="https://www.w3.org/TR/webvtt1/#default-text-color">https://www.w3.org/TR/webvtt1/#default-text-color</a>
<a href="https://www.w3.org/TR/webvtt1/#default-text-background">https://www.w3.org/TR/webvtt1/#default-text-background</a>

I added new default css classes with proper values for cues.
This change fixes problem with ignored color of subtitles
(and its background) delivered as WebVTT.

Reviewed by Eric Carlson.

* Source/WebCore/css/mediaControls.css:
(video::cue(.white)):
(video::cue(.lime)):
(video::cue(.cyan)):
(video::cue(.red)):
(video::cue(.yellow)):
(video::cue(.magenta)):
(video::cue(.blue)):
(video::cue(.black)):
(video::cue(.bg_white)):
(video::cue(.bg_lime)):
(video::cue(.bg_cyan)):
(video::cue(.bg_red)):
(video::cue(.bg_yellow)):
(video::cue(.bg_magenta)):
(video::cue(.bg_blue)):
(video::cue(.bg_black)):

Canonical link: <a href="https://commits.webkit.org/252464@main">https://commits.webkit.org/252464@main</a>
</pre>

eric-carlson commented on Source/WebCore/css/mediaControls.css:
```diff
@@ -177,6 +177,70 @@ video::cue {
 }
 
+video::cue(.white) { 
+    color: rgba(255,255,255,1);
```
Two minor nits: 
- none of these colors has alpha so you could use `rgb(...)` instead of `rgba(..., 1)`
- WebKit style is to put a space between comma separated values, so this should be `rgb(255, 255, 255)`

przemyslaw-gorszkowski-red replied:
With fixes after review I should do : 
git commit -a
Tools/Scripts/git-webkit pull-request
?

eric-carlson replied:
My own workflow for followup patches is:
- `git commit --amend path/to/new/changes`
- `git webkit pr --overwrite`

eric-carlson reviewed (approved):
Thank you for the fix!

r=me with two minor style change suggestions

pgorszkowski committed:
WebVTT: Add missing default css style for text and backgound colors.
https://bugs.webkit.org/show_bug.cgi?id=238540

Base on specification from:
https://www.w3.org/TR/webvtt1/#default-text-color
https://www.w3.org/TR/webvtt1/#default-text-background

I added new default css classes with proper values for cues.
This change fixes problem with ignored color of subtitles
(and its background) delivered as WebVTT.

Reviewed by Eric Carlson.

* Source/WebCore/css/mediaControls.css:
(video::cue(.white)):
(video::cue(.lime)):
(video::cue(.cyan)):
(video::cue(.red)):
(video::cue(.yellow)):
(video::cue(.magenta)):
(video::cue(.blue)):
(video::cue(.black)):
(video::cue(.bg_white)):
(video::cue(.bg_lime)):
(video::cue(.bg_cyan)):
(video::cue(.bg_red)):
(video::cue(.bg_yellow)):
(video::cue(.bg_magenta)):
(video::cue(.bg_blue)):
(video::cue(.bg_black)):

Canonical link: https://commits.webkit.org/252464@main
